templates/boot/generic-qemu-boot-template.jinja2

```diff
@@ -50,6 +54,11 @@ actions:
       minutes: 5
     method: qemu
     media: tmpfs
+{%- if qemu_docker %}
+    docker:
+      image: montjoie/qemu:bullseye
```

Comment:
```suggestion
      image: kernelci/qemu
```
Same thing as for the `kernelci/fastboot` Docker image, this can be changed to `kernelci/staging-qemu` to run tests on staging.